backend/server/controllers/user.js

```diff
@@ -31,8 +32,49 @@ export const getUsersById = async (req, res) => {
   try {
     const { params } = req;
     const user = await models.users.findByPk(params.id);
-    res.status(200).send(user);
+    const authToken = req.get('authToken');
```

Comment:
The authorization header will be under "Authorization', so it should be `req.get('Authorization)`